connect/eaas/core/extension.py

```diff
@@ -149,3 +150,37 @@ def setup_anvil_callables(self):
                 fn = functools.partial(_invoke, value)
                 fn.__name__ = value.__name__
                 anvil.server.callable(fn)
+
+
+class TransformationBase(ApplicationBase):
```

Comment:
No need to inherit from ApplicationBase.
The Runner will handle the set of transformation starting from the module so the way he get the extension.json descriptor will be starting from the module location.